meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn

```diff
@@ -4,13 +4,13 @@ source /usr/sbin/resin-vars
 VPN_PORT=443
 
 if [ ! -f $CONFIG_PATH ]; then
-    echo "$CONFIG_PATH : No such file."
+    echo "prepare-openvpn: $CONFIG_PATH - No such file."
     exit 1
 fi
 
 while true; do
-	if [ `jq ".uuid | length" $CONFIG_PATH` -eq 0 ]; then
-		echo "UUID missing from config file, VPN cannot connect"
+	if [ `jq ".uuid | length" $CONFIG_PATH` -eq 0 ] || [ `jq ".apiKey | length" $CONFIG_PATH` -eq 0 ]; then
```

Comment:
Now that we will generate uuid locally should we still wait for its generation? We kinda assume it now.